src/main/java/parser/Parser.java

```diff
@@ -0,0 +1,64 @@
+package parser;
+
+import command.*;
+import dukeexception.*;
+import task.*;
+
+
+public class Parser {
+    public static Command parseCommand(String userInput) throws DukeException{
+        String[] split = userInput.split(" ", 2);
+        String first = split[0];
+        switch (first) {
+            case "bye":
+                return new ExitCommand();
+
+            case "done":
+                int doneId = Integer.parseInt(split[1]);
```

Comment:
What if the user inputs a non-positive integer (ie. negative number, zero, double, or worse, a string)? From what I see now, there is currently no error handling for such cases (potential exceptions thrown: NumberFormatException if a string is parsed, ArrayIndexOutOfBoundsException if the integer is not within the range of indexes of the list). Perhaps you could consider looking into handling such cases.

This applies for Delete as well.